docs/get_started.md

```diff
@@ -110,32 +110,51 @@ the creation of a new management pack project.
     Along with additional language-specific configuration.
     Currently, `mp-init` offers the following languages:
 
-    `Python`
+    - `Python`
  
-    `Java` 
+    - `Java` 
+ 
+9.  === "Python"
+ 
+        a. `Select a template for your project`
 
-9. `(For Java projects only) Enter package name: `
+           Both of the available options will generate a project structure that can be modified into an adapter:
+
+           `Sample Adapter`: Creates a template adapter that collects several objects and metrics from the container the adapter is running.
+           The template adapter has comments throughout its code that explain what the code does and how to customize it for your adapter.
+
+           `New Adapter`: Creates methods, minimal comments, and code necessary to implement test
+           connection, collection, adapter definition, and endpoints logic.
 
-    The package name will be used to set up the package used by the adapter and the directory structure of the project.
-   
-    ???+ example 
- 
-        ``` java
-        package com.mycompany;
-        ```
 
-10. `Select a template for your project`
+    === "Java"
 
-    Both of the available options will generate a project structure that can be modified into an adapter:
+        a. `Enter package name: `
 
-    `Sample Adapter`: Creates a template adapter that collects several objects and metrics from the container the adapter is running.
-    The template adapter has comments throughout its code that explain what the code does and how to customize it for your adapter.
+           The package name will be used to set up the package used by the adapter and the directory structure of the project.
+           
+        ???+ example 
+     
+            ``` java
+            package com.mycompany;
+            ```
```

Comment:
Having an example here is nice, but I think it may be better to not have the java code and instead have the text we input for the question. If someone literally entered `package com.mycompany;` for this question, it wouldn't work.

Reply:
Actually, surprisingly `mp-init` doesn't complain, but you end up with this:

![image](https://github.com/vmware/vmware-aria-operations-integration-sdk/assets/3310870/7fc21f84-842b-4a53-a7fa-757bdfcb2002)